packages/suite/src/components/onboarding/SkipStepConfirmation.tsx

```diff
@@ -1,38 +1,55 @@
+import { ReactNode } from 'react';
 import * as STEP from 'src/constants/onboarding/steps';
 import { AnyStepId } from 'src/types/onboarding';
 import { NewModal } from '@trezor/components';
 import { Translation } from 'src/components/suite';
 import { useOnboarding } from 'src/hooks/suite';
 
-interface SkipStepConfirmationProps {
+type SkipStepConfirmationProps = {
     onCancel: () => void;
-}
+};
 
-export const SkipStepConfirmation = ({ onCancel }: SkipStepConfirmationProps) => {
-    const { activeStepId, goToNextStep } = useOnboarding();
+type SkipModalContent = {
+    heading?: ReactNode;
+    cta?: ReactNode;
+    body?: ReactNode;
+    nextStep?: AnyStepId;
+};
 
-    let text;
-    let nextStep: AnyStepId;
+const getModalContent = (activeStepId: AnyStepId): SkipModalContent => {
     switch (activeStepId) {
         case STEP.ID_SECURITY_STEP:
         case STEP.ID_BACKUP_STEP:
-            text = <Translation id="TR_SKIP_BACKUP" />;
-            nextStep = STEP.ID_SET_PIN_STEP;
-            break;
+            return {
+                heading: <Translation id="TR_SKIP_BACKUP" />,
+                cta: <Translation id="TR_SKIP_BACKUP" />,
```

Comment:
I believe this shouldn't be call to action. WDYT?

<img width="635" alt="Screenshot 2024-10-08 at 11 43 52" src="https://github.com/user-attachments/assets/4f49273f-2a81-4758-8353-16beafa65b0e">

Reply:
I know, I just didn't want to think too much about the naming:D

Reply:
What about `secondaryButtonText`